src/components/ImageDetails/RelatedImages.vue

```diff
@@ -9,9 +9,10 @@
     <ImageGrid
       :images="images"
       :can-load-more="false"
-      :is-fetching="$fetchState.pending"
-      :fetching-error="$fetchState.error"
-      :error-message-text="null"
+      :fetch-state="{
+        isFetching: $fetchState.pending,
+        fetchingError: $fetchState.error,
+      }"
```

Comment:
I really like this!

Reply:
How does this work with reactivity? Is it working as a side-effect of the object being redeclared on each render?